ui/src/assets/docs/de/developer-portal-page.md

```diff
@@ -0,0 +1,364 @@
+# Developer Portal
+
+---
+
+## Einleitung
+
+Die [Payment Service Directive 2 (PSD2)](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32015L2366&from=EN)
+schreibt Banken einen einheitlichen Standard für eine Account-Schnittstelle erreichbar für Drittanbieter vor. Im weiteren Text wird diese Account-Schnittstelle als "XS2A" bezeichnet, was für Acces-to-Acount" steht. <!-- vielleicht wäre ein Glossar hilfreich? -->  
+Der Produktivgang dieser Live-Schnittstelle ist für September 2019 vorgeschrieben. XS2A
+setzt sich aus folgende Banking-Funktionalitäten: Zahlungsinitiierung (PIS=Payment Initiation Service), Kontodatenabfrage (AIS=Account Information Service) und Abfrage von Zahlungsfähigkeit (FCS=Funds Confirmation Service).
+Um die Einhaltung der Frist zu gewährleisten und Fehlerbehebungen und Anpassungen ebenfalls noch berücksichtigen zu können, ist es in PSD2 vorgeschrieben eine Testversion der Schnittstelle bereits im März 2019 zur Verfügung zu stellen.
+
+Zentrale Komponente des PSD2 Accelerator ist die XS2A-Schnittstelle, die auf gemäß der Spezifikation der [Berlin Group](https://www.berlin-group.org/)
+(Version 1.3) implementiert wurde und auf Testdaten basiert. Neben der Schnittstelle ist es für Banken ebenfalls vorgeschrieben eine kostenfreie technische Dokumentation bereitzustellen, die über unterstützte Produkte und Zahlungsservices informiert.
+
+Zukünftig wird es für einen Drittanbieter notwendig sein sich bei der BAFIN zu registrieren und sich von einem Treuhanddienstleister (TSP=Trust Service Provider) ein [eIDAS](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32014R0910&from=EN) Zertfikat ausstellen zu lassen, um XS2A-Funktionalitäten bei Banken nutzen zu können. 
+Da das Ausstellen eines solches Zertifikates für Testzwecke einen sehr hohen Aufwand darstellen würde, wird im Rahmen des adorsys PSD2 Accelerator ein TSP simuliert und eine Funktion zur Erstellung eines qualifizierten Website Authentifizierungs-Zertifikates (=QWAC) bereitgestellt. Ein QWAC ist Teil der eIDAS und wird auch als [X.509](https://www.ietf.org/rfc/rfc3739.txt)
+Zertifikat bezeichnet. Für PSD2-Zwecke wird das Zertfikat 
+<!-- mittels QcStatement Was bedeutet das, wie soll man es übersetzen?! -->
+um Inhalte wie die Rolle des Drittanbieters erweitert (Siehe auch [ETSI](https://www.etsi.org/deliver/etsi_ts/119400_119499/119495/01.01.02_60/ts_119495v010102p.pdf)).
+
+Mit der Integration des QWAC in einem XS2A Request wird die Rolle und Signatur an einem zentralen Reverse Proxy geprüft, bevor die Anfrage an die XS2A-Schnittstelle weitergereicht wird, in der die Banklogik abgebildet wird. Unsere Dokumentation finden Sie hier: [XS2A Swagger UI](/swagger-ui.html).
+
+Die beschriebenen Komponenten und das Zusammenspiel dieser, werden in Abbildung 1.1 dargestellt:
```

Comment:
Die beschriebenen Komponenten und ihr Zusammenspiel werden in Abbildung 1.1 dargestellt:

Reply:
done